app.py

```diff
@@ -59,4 +61,5 @@ def index():
 
 if __name__ == "__main__":
     app = create_app()
-    app.run(host=os.environ['FLASK_RUN_HOST'], ssl_context='adhoc')
+    #app.run(host=os.environ['FLASK_RUN_HOST'], ssl_context='adhoc')
+    app.run(ssl_context='adhoc')
```

Comment:
Pourquoi ce changement ?

Reply:
Sinon ça marche pas comme j'ai pas docker :(

Reply:
you should not commit this